pkg/tfbridge/detailed_diff.go

```diff
@@ -427,16 +428,63 @@ func (differ detailedDiffer) makePropDiff(
 	}
 }
 
+func makeListAttributeDiff(
+	path propertyPath, old, new []resource.PropertyValue,
+) map[detailedDiffKey]*pulumirpc.PropertyDiff {
```

Comment:
Can we add a comment here that (1) `makeListAttributeDiff` should only be called on lists of scalar values (not collections) and (2) add some asserts to the same effect.